icons/plus-flash.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path stroke-width="1.08106" d="M8.359 11.817v7.568M4.575 15.601h7.568"/>
+    <g fill="#000" stroke="none">
```

Comment:
You should use strokes, _not_ fills. Icons should always have an outline style.

Reply:
Ok, I'll fix this